setup.py

```diff
@@ -63,26 +63,26 @@ def initialize_options(self):
         # Default to /usr/local for Homebrew
         prefix = '/usr/local'
       else:
-        print ('Warning: mfg-inspector output is not fully implemented for '
-               'Windows. OpenHTF will be installed without it.')
+        print('Warning: mfg-inspector output is not fully implemented for '
+              'Windows. OpenHTF will be installed without it.')
         self.skip_proto = True
 
-    self.protoc = os.path.join(prefix, 'bin', 'protoc')
-    self.protodir = os.path.join(prefix, 'include')
-    self.indir = os.path.join(os.getcwd(), 'openhtf', 'output', 'proto')
-    self.outdir = os.path.join(os.getcwd(), 'openhtf', 'output', 'proto')
+    self.protoc = os.path.join(prefix, b'bin', b'protoc')
+    self.protodir = os.path.join(prefix, b'include')
+    self.indir = os.getcwd()
+    self.outdir = os.getcwd()
 
   def finalize_options(self):
     pass
 
   def run(self):
     if self.skip_proto:
-      print 'Skipping building protocol buffers.'
+      print('Skipping building protocol buffers.')
       return
     # Build regular proto files.
-    protos = glob.glob(os.path.join(self.indir, '*.proto'))
+    protos = glob.glob(os.path.join(self.indir, 'openhtf', 'output', 'proto', '*.proto'))
```

Comment:
Maintainers: this is what I mentioned above. Since proto imports are relative to the project root and not their specific location, it seemed correct to invoke the proto compiler from the root as well.

Note that this doesn't change the strategy or location of the pb2.py files, it just aligns the new import path requirement with the compilation paths.

Reply:
SGTM